models/gc_spider_baseline/utils/SpiderBaselineRunner.py

```diff
@@ -0,0 +1,194 @@
+        'spider_baseline_vertebrae_segmentation_raw.mha',
+        'mha:mod=seg:model=SpiderBaselineSegmentation:seg=raw',
+        data='in_data',
+        bundle='model',
+        the='Spider baseline vertebrae segmentation for the input sagittal spine MRI or CT. Raw segmentation output: '
+            '0: background '
+            '1-24: different vertebrae numbered from the bottom (i.e. L5 = 1, C1 = 24) '
+            '100: spinal canal '
+            '101-124 for partially visible vertebrae '
+            '201-223: different intervertebral discs (also numbered from the bottom, i.e. L5/S1 = 201, C2/C3 = 223)'
+    )
+    @IO.Output(
+        'out_data',
+        'spider_baseline_vertebrae_segmentation.mha',
+        'mha:mod=seg:model=SpiderBaselineSegmentation:seg=remapped:roi=VERTEBRAE_L5,VERTEBRAE_L4,VERTEBRAE_L3,VERTEBRAE_L2,'
+        'VERTEBRAE_L1,VERTEBRAE_T12,VERTEBRAE_T11,VERTEBRAE_T10,VERTEBRAE_T9,VERTEBRAE_T8,VERTEBRAE_T7,VERTEBRAE_T6,VERTEBRAE_T5,'
+        'VERTEBRAE_T4,VERTEBRAE_T3,VERTEBRAE_T2,VERTEBRAE_T1,VERTEBRAE_C7,VERTEBRAE_C6,VERTEBRAE_C5,VERTEBRAE_C4,VERTEBRAE_C3,VERTEBRAE_C2,VERTEBRAE_C1,'
+        'VERTEBRAE_DISK_L5S1,VERTEBRAE_DISK_L4L5,VERTEBRAE_DISK_L3L4,VERTEBRAE_DISK_L2L3,VERTEBRAE_DISK_L1L2,VERTEBRAE_DISK_T12L1,VERTEBRAE_DISK_T11T12,'
+        'VERTEBRAE_DISK_T10T11,VERTEBRAE_DISK_T9T10,VERTEBRAE_DISK_T8T9,VERTEBRAE_DISK_T7T8,VERTEBRAE_DISK_T6T7,VERTEBRAE_DISK_T5T6,VERTEBRAE_DISK_T4T5,'
+        'VERTEBRAE_DISK_T3T4,VERTEBRAE_DISK_T2T3,VERTEBRAE_DISK_T1T2,VERTEBRAE_DISK_C7T1,VERTEBRAE_DISK_C6C7,VERTEBRAE_DISK_C5C6,VERTEBRAE_DISK_C4C5,'
+        'VERTEBRAE_DISK_C3C4,VERTEBRAE_DISK_C2C3,'
+        'SPINAL_CANAL',
+        data='in_data',
+        bundle='model',
+        the='Spider baseline vertebrae segmentation for the input sagittal spine MRI or CT. Remapped segmentation output: '
+            '0: background '
+            '1-24: different vertebrae numbered from the bottom (i.e. L5 = 1, C1 = 24) (includes partially visible vertebrae from raw segmentation) '
+            '25-47: different intervertebral discs (also numbered from the bottom, i.e. L5/S1 = 25, C2/C3 = 47) '
+            '48: spinal canal'
+    )
+    def task(self, instance: Instance, in_data: InstanceData, out_data_raw: InstanceData, out_data: InstanceData) -> None:
+        if in_data.type.meta <= MR:
+            config = self.setup_mr()
+        elif in_data.type.meta <= CT:
+            config = self.setup_ct()
+        else:
+            raise ValueError(f"SpiderBaselineRunner does not support other modalities than CT and MR")
+        self.process(
+            in_data=in_data,
+            out_data_raw=out_data_raw,
+            **config
+        )
+        self.create_remapped_segmentation(out_data_raw, out_data)
+
+    def create_remapped_segmentation(self, out_data_raw: InstanceData, out_data: InstanceData):
+        # Create the remapping dictionary to reorder output labels so they will be picked up in the correct order by DicomSeg
```

Comment:
'so they will be picked up in the correct order by DicomSeg' -> 'to match with the annotation in the roi meta parameter', right?

Reply:
yes, that's correct. I'll rephrase it.